app/Kconfig

```diff
@@ -410,6 +410,14 @@ config ZMK_COMBO_MAX_KEYS_PER_COMBO
 #Combo options
 endmenu
 
+menu "Behavior Options"
+
+config ZMK_BEHAVIORS_QUEUE_SIZE
+	int "Maximum number of behaviors to allow queueing from a macro or other complex behavior"
+	default 20
```

Comment:
Should we default this to a number larger than we expect most people to ever use, like 64/128? Boards with less memory can turn it down as needed.

Otherwise, we're going to run into problems when we add things like a send string behavior and someone tries to send a string more than 20 characters long.